python/paddle/v2/fluid/layers/nn.py

```diff
@@ -1434,29 +1433,24 @@ def sequence_last_step(input):
 def pool2d(input,
            pool_size,
```

Comment:
The pool_size also needs to be set None by default, since if set `global_pooling` = True, there is no need to set `pool_size`.

Reply:
Done